dynawo/sources/Modeler/DataInterface/DYNComponentInterface.cpp

```diff
@@ -101,6 +101,19 @@ ComponentInterface::getStateVariableReference() {
   }
 }
 
+void
+ComponentInterface::backupComponentVar(const std::string& varName) {
+  int index = getComponentVarIndex(varName);
+  if (index > 0)
+    stateVariables_[index].setBackupValue();
+}
+
+bool
+ComponentInterface::hasComponentVarChanged(const std::string& varName, const double fromValue) const {
+  int index = getComponentVarIndex(varName);
+  return (index > 0) && stateVariables_[index].hasValueChanged(fromValue);
```

Comment:
Why is index=0 excluded?

Reply:
This is a bug. Same in backupComponentVar.
Fixed !